dm/syncer/syncer.go

```diff
@@ -3075,10 +3075,15 @@ func (s *Syncer) loadTableStructureFromDump(ctx context.Context) error {
 				setFirstErr(err)
 				continue
 			}
-			s.saveTablePoint(
-				&filter.Table{Schema: db, Name: stmtNode.(*ast.CreateTableStmt).Table.Name.O},
-				s.getFlushedGlobalPoint(),
-			)
+			switch v := stmtNode.(type) {
+			case *ast.CreateTableStmt:
+				s.saveTablePoint(
+					&filter.Table{Schema: db, Name: stmtNode.(*ast.CreateTableStmt).Table.Name.O},
```

Comment:
```suggestion
					&filter.Table{Schema: db, Name: v.Table.Name.O},
```

I remember we can `v` like this.